### PR TITLE
Replace &amp; by & as parameter query divider

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
## Type
- Critical bugfix

## Pull request description
Using &amp; will resolve in the following queryString: index?foo=test&amp;bar=test&amp;token=x - which will resolve in the following parameters: foo = test, amp;=bar = test, amp;token = x
Replacing it by & will resolve this issue.

### CODE REVIEW

File: src/Backend/Core/Engine/Model.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public static function createURLForAction(
             $parameters['sort'] = $queryParameterBag->get('sort');
         }
 
-        $queryString = '?' . http_build_query($parameters, null, '&amp;');
+        $queryString = '?' . http_build_query($parameters, null, '&');
```
Comment (carakas) [inline src/Backend/Core/Engine/Model.php]: `&` is the default so I think you can just use `http_build_query($parameters)`

Comment (jonasdekeukelaere) [inline src/Backend/Core/Engine/Model.php]: Fixed ecf455d

**Comment by jonasdekeukelaere** [comment]
@carakas Also found some more uses of &amp; instead of & in other classes.

**Comment by carakas** [comment]
@jonasdekeukelaere did you test all the usecases that this doesn't give issues in the template?

**Comment by jonasdekeukelaere** [comment]
I didn't test all use cases, but I tested it where it went wrong (contentblocks - report after editing/deleting was incorrect) and works. Also checked the rss (frontend) where the frontend fix is being used. In both cases the url is now correct.

**Pull request merged**